### ISSUE TITLE
note in msItemStruct

### ISSUE THREAD

**Issue by atoboy** [djbpitt/repertorium#9]
We have various notes as children of `<msItemStruct>` element. Some of them do not have attributes at all, some of them have various encoding, e.g. AA1322NBKM:
```xml
 <note place="foot" xml:id="note1">The text presents the type of Thunderbook,
                            predicting by the Zodiac circle. ...</note>
```
or from the same description:
```xml
<note place="foot" type="articleData" xml:id="note2">The text is wrongly
                            titled as Lunary. ...</note>
```
In most of the descriptions `<note>` in this context has no attributes at all.

Suggestion:
Use `<note>` without any attributes. When it is a child of `<msItemStruct>` display it as paragraph, not as footnote. 

It will be good if the content of `<note>` is presented in HTML edition of the description.

**Comment by djbpitt** [comment]
@atoboy I agree with your proposal. I'll move this Issue to the **To be implemented** column in the project board so that you can modify the ODD and adapt the existing manuscript description files. Once you've done that, if you can move the card to the **Update documentation** column, I'll update our documentation addendum file at http://repertorium.obdurodon.org/encoding-policies-update.xhtml and close the issue.
